### ISSUE TITLE
Phone number exercise contains test cases w/ invalid phone numbers

### ISSUE THREAD

**Issue by porkostomus** [exercism/clojure#300]
From the README:

```
The format is usually represented as
(NXX)-NXX-XXXX
where N is any digit from 2 through 9 and X is any digit from 0 through 9.
```

But there are test cases with area codes that start with 1:

```
(deftest cleans-number
  (is (= "1234567890" (phone-number/number "(123) 456-7890"))))
```

It appears that the other tracks don't have this issue; see https://github.com/exercism/csharp/blob/master/exercises/phone-number/PhoneNumberTests.cs

**Comment by Grazfather** [comment]
Looks like this is fixed in #301

**Comment by haus** [comment]
good call @Grazfather . closing this out.
